paseos/actors/base_actor.py

```diff
@@ -33,37 +34,38 @@ class BaseActor(ABC):
     _central_body = None
 
     # Communication links dictionary
-    _communication_links = DotMap(_dynamic=False)
+    _communication_devices = DotMap(_dynamic=False)
 
-    def __init__(
-        self, name: str, position, velocity, epoch: pk.epoch, central_body: pk.planet
-    ) -> None:
+    def __init__(self, name: str, position, epoch: pk.epoch) -> None:
         """Constructor for a base actor
 
         Args:
             name (str): Name of this actor
             position (list of floats): [x,y,z]
-            velocity (list of floats): [vx,vy,vz]
             epoch (pykep.epoch): Epoch at this pos / velocity
-            central_body (pk.planet): pykep central body
         """
         logger.trace("Instantiating Actor.")
-        BaseActor._check_init_value_sensibility(position, velocity)
         super().__init__()
         self.name = name
         self._local_time = epoch
-        self._central_body = central_body
-        self._orbital_parameters = pk.planet.keplerian(
-            epoch,
-            position,
-            velocity,
-            central_body.mu_self,
-            1.0,
-            1.0,
-            1.0,
-            name,
-        )
-        self._communication_links = DotMap(_dynamic=False)
+
+        self._communication_devices = DotMap(_dynamic=False)
+
+    def get_local_time(self) -> pk.epoch:
+        """Returns local time of the actor as pykep epoch. Use e.g. epoch.mjd2000 to get time in days.
+
+        Returns:
+            pk.epoch: local time of the actor
+        """
+        return self._local_time
+
+    def get_communication_devices(self) -> DotMap:
```

Comment:
same as above

Reply:
I presume you meant the property stuff, right? Because that's the above file for me atm

Reply:
Yes, sorry for the ambiguity.